libkbfs/tlf_handle.go

```diff
@@ -67,6 +67,16 @@ func (h TlfHandle) IsReader(user keybase1.UID) bool {
 	return ok
 }
 
+// ResolvedReadersMap returns a map of resolved readers from uid to usernames.
+func (h TlfHandle) ResolvedReadersMap() map[keybase1.UID]libkb.NormalizedUsername {
```

Comment:
Can you change this to `ResolvedUsersMap` and build a new map with both readers/writers? That way, we get the nice side effect of not exposing the internal map of the handle, and simplifying the callers a bit.